File: HackerRank Solutions/10 Days of Statistics/Day 4 Geometric Distribution I.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
in0 = list( map( float, input().strip().split(" ") ) )
n = int(input())

pf = in0[0] / in0[1]

p = (1-pf)**(n-1) * pf

print("%.3f" % p)

File: HackerRank Solutions/10 Days of Statistics/Day 4 Geometric Distribution I.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */ 
    float num, deno, try;
    float def, prob;
    scanf("%f %f", &num, &deno);
    scanf("%f", &try);
    def = num / deno;
    prob = pow(1-def, (try-1))*def;
    printf("%.3f", prob);
    return 0;
}